db/system_keyspace.cc

```diff
@@ -36,6 +36,7 @@
 #include "gms/feature_service.hh"
 #include "system_keyspace_view_types.hh"
 #include "schema/schema_builder.hh"
+#include "schema/schema_registry.hh"
```

Comment:
I expected (after reading the more detailed description) that we'd see the source as an individual sstable, not "sstable" as a class. I guess it can be added as needed.

Reply:
See above.